README.md

```diff
@@ -19,6 +19,8 @@ This package allows you to download, export and import your application's databa
 - User authentication through Laravel Sanctum tokens
 - Transport encryption using Sodium
 
+## Notes
+- Enabling Laravel Telescope will prevent remote files from being downloaded, as it opens and discards the HTTP stream!
```

Comment:
Are there any details on why this happens?
Sounds like something that should be fixed, either within this package or Telescope.

Reply:
its because telescope inspects the stream. ideas on how to prevent it are welcome

Reply:
https://github.com/cybex-gmbh/laravel-protector/pull/43